README.md

```diff
@@ -242,6 +242,32 @@ module "iam_group_with_policies" {
 }
 ```
 
+`iam-eks-role`:
+
+```hcl
+module "iam_eks_role" {
+  source      = "terraform-aws-modules/iam/aws//modules/iam-eks-role"
+  version     = "~> 4"
```

Comment:
Use a newline to separate meta-arguments from the rest. Put a newline after `version`.

Reply:
Done!